### ISSUE TITLE
Mint NFT after lesson completion

### ISSUE THREAD

**Issue by Markkos89** [Developer-DAO/academy-turbo#205]

**Comment by kempsterrrr** [comment]
This is a very valuable feature for grants and partnerships. It would be especially valuable if we were able to offer this feature across next woks. By this I mean we can deploy the NFTs to different chains depending on the course - this is a common feature on grant proposal I have seen in the space. What funders are looking for is chain usage and this really helps by ensuring your activating a new user on the chain and deploying contacts that the generate activity .

**Comment by Markkos89** [comment]
There is a starting notion page with `Krish` (DD Member) willing to develop the feature (Solidity) Notion page: https://developerdao.notion.site/Academy-NFTs-Create-soul-bound-NFT-for-a-lesson-completion-44434c4295194d9ea10bed99525d394a 

If any further details or desired requirements regarding NFT certification appears, it would be nice to have them there. So far:
- Souldbound NFT
- Multi-chain
- Lesson completion
- Track completion

**Comment by kempsterrrr** [comment]
Thanks for this @Markkos89 ❤️  - tagging the other relevant issues as I almost cross posted between them #206 #209 #212 

Awesome someone is thinking about this already. Have you considered any platforms that might support this and make it a lot easier to implement? There is a lot of infrastructure to build to create this feature set well, I think considering something like [Syndicate](https://syndicate.io/) is a sensible approach as they provide what we need across all EVM chains in a flexible way out of the box:

- Pre-audited smart contracts
- Transaction broadcasting
- Sponsors transaction and gas management

Having these features out of the box like this would really improve the team + projects velocity.

**Comment by kempsterrrr** [comment]
I was looking into this more this evening and realise Syndicate doesn't support enough networks, [Thirdweb](https://thirdweb.com/) seems like it would be a far better option. @Markkos89 would you consider building a platform like Thirdweb vs building everything in-house? Might help simplify the build particularly with a small team.

**Comment by Markkos89** [comment]
> I was looking into this more this evening and realise Syndicate doesn't support enough networks, [Thirdweb](https://thirdweb.com/) seems like it would be a far better option. @Markkos89 would you consider building a platform like Thirdweb vs building everything in-house? Might help simplify the build particularly with a small team.

Sorry @kempsterrrr I don't understand very well the sentence.. you mean to actually build a platform like Thirdweb? 
Or you are saying to build this features **using** thirdweb?

The first option's answer would be a _no, I don't consider to build a platform like Thirdweb_
In case is the second option then **yes**, I can use Thirdweb to accomplish this task, but I'd prefer to help with the certification logic and delegate this task to someone with more Solidity knowledge, like for example Krish, which was interested in contribute with this development.

Thanks mate, let me know if the answer helps or if something it's unclear :)

**Comment by kempsterrrr** [comment]
Yes, mate, I meant option 2: use a platform like Thirdweb. I totally agree with you on delegating to someone with solidity knowledge; one of the good things about using such a platform is that they have audited smart contracts ready to go, which also means we get a nice admin UI for free for the contracts.

**Comment by elPiablo** [comment]
Thirdweb is a really nice interface. We use if for mentorship. We mint on Polygon, and added in a couple of extra parameters that 'we might need in future cases', which we ended up using when we updated the NFT with some extra details. Really nice set of tools.
